docs/pages/installation/eks.md

```diff
@@ -9,32 +9,41 @@ list: include
 image: /assets/img/platforms/eks.png
 ---
 
+Manage your EKS clusters with Meshery. Deploy Meshery on EKS or outside of EKS.
 {% include installation_prerequisites.html %}
 
-## Set up and run Meshery on EKS:
+### General Prerequisites:
```

Comment:
Let's not have two Prerequisites sections. Instead include these prereqs in the existing callout section.